apps/tsilhqotin-language-hub/src/Components/Pages/Apps/AppsDetail.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+
+export function AppsDetail() {
+    const appData = [
```

Comment:
Great first step of creating a data array. Let's see if we can go a step further and load this from a config file. This will make the code completely reusable by other groups or for other projects.

Reply:
We fixed this.